src/Storages/ObjectStorage/DataLakes/IcebergMetadata.cpp

```diff
@@ -82,59 +86,38 @@ enum class DataFileContent : uint8_t
     EQUALITY_DELETES = 2,
 };
 
-/**
- * Iceberg supports the next data types (see https://iceberg.apache.org/spec/#schemas-and-data-types):
- * - Primitive types:
- *   - boolean
- *   - int
- *   - long
- *   - float
- *   - double
- *   - decimal(P, S)
- *   - date
- *   - time (time of day in microseconds since midnight)
- *   - timestamp (in microseconds since 1970-01-01)
- *   - timestamptz (timestamp with timezone, stores values in UTC timezone)
- *   - string
- *   - uuid
- *   - fixed(L) (fixed-length byte array of length L)
- *   - binary
- * - Complex types:
- *   - struct(field1: Type1, field2: Type2, ...) (tuple of typed values)
- *   - list(nested_type)
- *   - map(Key, Value)
- *
- * Example of table schema in metadata:
- * {
- *     "type" : "struct",
- *     "schema-id" : 0,
- *     "fields" : [
- *     {
- *         "id" : 1,
- *         "name" : "id",
- *         "required" : false,
- *         "type" : "long"
- *     },
- *     {
- *         "id" : 2,
- *         "name" : "array",
- *         "required" : false,
- *         "type" : {
- *             "type" : "list",
- *             "element-id" : 5,
- *             "element" : "int",
- *             "element-required" : false
- *     },
- *     {
- *         "id" : 3,
- *         "name" : "data",
- *         "required" : false,
- *         "type" : "binary"
- *     }
- * }
- */
+std::pair<size_t, size_t> parse_decimal(const String & type_name)
```

Comment:
Please, use `camelCase` names for functions. Also, place this function in an anonymous namespace as it's used only in this file:
```c++
namespace
{

std::pair<size_t, size_t> parsDecimal(const String & type_name)
{
...
}

}
```